midealocal/device.py

```diff
@@ -462,7 +468,11 @@ def update_all(self, status: dict[str, Any]) -> None:
 
     def enable_device(self, available: bool = True) -> None:
         """Enable device."""
-        _LOGGER.debug("[%s] Enabling device", self._device_id)
+        _LOGGER.debug(
+            "[%s] %s device",
+            self._device_id,
+            "Enabling" if available else "Disabling",
```

Comment:
Agree, but then function named should be changed

Reply:
This might cause major conflicts and lots of refactoring, as well as becoming a breaking change.

Reply:
True but still the name is misleading and I really don't think we should leave it like it is.
Once we find a wrong thing, we should always fix it